client/my-sites/plans/main.jsx

```diff
@@ -241,5 +248,6 @@ export default connect( ( state ) => {
 		showTreatmentPlansReorderTest: isTreatmentPlansReorderTest( state ),
 		plansLoaded: Boolean( getPlanSlug( state, getPlan( PLAN_FREE )?.getProductId() || 0 ) ),
 		is2023OnboardingPricingGrid,
+		isJetpackNotAtomic: isJetpackSite( state, selectedSiteId, { treatAtomicAsJetpackSite: false } ),
```

Comment:
Apparently, there's a `options.treatAtomicAsJetpackSite` parameter that can be set on this selector (and avoid having to [combine two selectors](https://github.com/Automattic/wp-calypso/pull/72960/files#diff-9a7bb16689c221cf314662dc7929cea29ae46431f9bc11fd4123e233b0b208d2R317-R318) for it as in the other file). Not sure I'm missing something though. @jdc91

Reply:
Nice this is a better approach!